tests/integration/test_jobs.py

```diff
@@ -71,3 +71,20 @@ def test_last_job_runs(w):
 
     for line in sorted(summary, key=lambda s: s['last_finished'], reverse=True):
         logging.info(f'Latest: {line}')
+
+
+def test_create_job(w):
+
+    from databricks.sdk.service.jobs import (BaseClusterInfo, JobCluster,
+                                             JobTaskSettings, PythonWheelTask)
+
+    cluster = JobCluster(job_cluster_key="cluster1",
+                         new_cluster=BaseClusterInfo(num_workers=2,
+                                                     spark_version="13.0.x-cpu-ml-scala2.12",
```

Comment:
use `w.clusters.select_spark_version`, because we don't want to depend on concrete DBR versions in the test code. See https://github.com/databricks/databricks-sdk-py/blob/main/examples/starting_job_and_waiting.py#L59-L71